src/main/java/net/snowflake/ingest/streaming/internal/ArrowFlusher.java

```diff
@@ -104,6 +103,6 @@ public Flusher.SerializationResult serialize(
       }
     }
     return new Flusher.SerializationResult(
-        channelsMetadataList, columnEpStatsMapCombined, rowCount);
+        channelsMetadataList, columnEpStatsMapCombined, rowCount, chunkData);
```

Comment:
Yes, this is the same for both.
I think it is better to have it as an explicit serialisation output instead of an input argument because it is produced in the serialize function.

Reply:
I agree with @sfc-gh-azagrebin.